x-pack/plugins/ml/public/application/jobs/jobs_list/components/job_actions/results.js

```diff
@@ -37,29 +44,59 @@ export function ResultLinks({ jobs }) {
   const singleMetricEnabled = jobs.length === 1 && jobs[0].isSingleMetricViewerJob;
   const jobActionsDisabled = jobs.length === 1 && jobs[0].deleting === true;
   const { createLinkWithUserDefaults } = useCreateADLinks();
+  const timeSeriesExplorerLink = useMemo(
+    () => createLinkWithUserDefaults('timeseriesexplorer', jobs),
+    [jobs]
+  );
+  const anomalyExplorerLink = useMemo(() => createLinkWithUserDefaults('explorer', jobs), [jobs]);
+
   return (
     <React.Fragment>
       {singleMetricVisible && (
         <EuiToolTip position="bottom" content={openJobsInSingleMetricViewerText}>
+          {isManagementTable ? (
+            <EuiButtonIcon
+              href={`${basePath.get()}/app/ml/${timeSeriesExplorerLink}`}
```

Comment:
Link should be created by ML URL generator

Reply:
I'll keep this here for now because these are being generated using `mlJobService.createResultsUrlForJobs` and we need both versions (with and without `basePath`) so even if we use the MlUrlGenerator, we will have to call this line.

Reply:
I believe it will make code easier to maintain having a single source of truth for URLs. We can inject `mlJobService` into the URL generator and invoke `createResultsUrlForJobs` from there.